docs/docs/en/guide/project/workflow-definition.md

```diff
@@ -128,6 +128,9 @@ The following are the operation functions of the workflow definition list:
 - **Timing Management:** The timing management page can edit, online or offline and delete timing.
 - **Delete:** Delete the workflow definition. In the same project, only the workflow definition created by yourself can be deleted, and the workflow definition of other users cannot be deleted. If you need to delete it, please contact the user who created it or the administrator.
 - **Download:** Download workflow definition to local.
+- **Copy:** Copy a new workflow based on the current workflow.
```

Comment:
Could we add some notice to tell users can copy to both some project or to others project?
BTW, we should also tell user when they copy the workflow, the workflow name will and copy date as is post to compare between previous tasks

Reply:
Thanks for your comment. I'll add this.